manager/assets/modext/widgets/element/modx.tree.element.js

```diff
@@ -563,6 +563,32 @@ Ext.extend(MODx.tree.Element,MODx.tree.Tree,{
             });
         }
 
+        if (n.isLoaded()) {
+            var childNodes = n.childNodes;
+
+            if (childNodes.some(function(child) { return !child.leaf; })) { // If any childNode has own children
+                m.push('-');
+    
+                if (n.isExpanded() && childNodes.some(function(child) { return child.isExpanded(); })) { // If any childNode is expanded
```

Comment:
`childNodes.some(function(child) { return child.isExpanded(); })`
VVV
`childNodes.some(child => child.isExpanded())`

Reply:
Useless comment (`// If any childNode is expanded`)

Reply:
> Useless comment (`// If any childNode is expanded`)

All 4 of your comments are useless